run_sweep.py

```diff
@@ -106,6 +106,7 @@ def _run_model_test(model_path: pathlib.Path, test: str, device: str, jit: bool,
         num_batches = task.get_model_attribute("NUM_BATCHES")
         if num_batches:
             result.results["latency_ms"] = result.results["latency_ms"] / num_batches
+        result.results["latency_ms"] = round(result.results["latency_ms"], 3)
```

Comment:
I think it is still good to have precise latency numbers here, what do you think?

Reply:
It's rational to keep the original data. How about changeing it in the summarize.py, so that we don't change the original data, and maker the output shorter.

Reply:
I think we can add a parameter such as pretty mode or max-precision when printing out the summary, and left the original data as it is.